python_scripts/integration_test_with_api.py

```diff
@@ -248,19 +245,19 @@ def generate_destination_plate_barcode() -> str:
 def generate_baracoda_barcodes(count, prefix) -> list:
     # Current accepted prefixes: https://github.com/sanger/baracoda/blob/4f001ecc771311ec85c07ea5f5e67e4c313adef5/tests/data/fixture_data.py
 
-    baracoda_url = f"http://{BARACODA_URL}/barcodes_group/{prefix}/new?count={count}"
+    baracoda_url = f"https://{BARACODA_URL}/barcodes_group/{prefix}/new?count={count}"
 
     try:
         print(f"Sending request for destination barcode to: {baracoda_url}")
-        response = requests.post(baracoda_url, data={})
+        response = requests.post(baracoda_url, data={}, verify=False) # Never use verify=False apart from in testing
```

Comment:
To make it work with verify=True when run with Sanger computers, have you tried defining REQUESTS_CA_BUNDLE? Like in here:

https://stackoverflow.com/questions/42982143/python-requests-how-to-use-system-ca-certificates-debian-ubuntu